mdata/aggmetric.go

```diff
@@ -31,39 +31,47 @@ type AggMetric struct {
 	store       Store
 	cachePusher cache.CachePusher
 	sync.RWMutex
-	key             schema.AMKey
-	rob             *ReorderBuffer
-	currentChunkPos int    // Chunks[CurrentChunkPos] is active. Others are finished. Only valid when len(chunks) > 0, e.g. when data has been written (excl ROB data)
-	numChunks       uint32 // max size of the circular buffer
-	chunkSpan       uint32 // span of individual chunks in seconds
-	chunks          []*chunk.Chunk
-	aggregators     []*Aggregator
-	dropFirstChunk  bool
-	ttl             uint32
-	lastSaveStart   uint32 // last chunk T0 that was added to the write Queue.
-	lastSaveFinish  uint32 // last chunk T0 successfully written to Cassandra.
-	lastWrite       uint32 // wall clock time of when last point was successfully added (possibly to the ROB)
-	firstTs         uint32 // timestamp of first point seen
+	key                    schema.AMKey
+	rob                    *ReorderBuffer
+	currentChunkPos        int    // Chunks[CurrentChunkPos] is active. Others are finished. Only valid when len(chunks) > 0, e.g. when data has been written (excl ROB data)
+	numChunks              uint32 // max size of the circular buffer
+	chunkSpan              uint32 // span of individual chunks in seconds
+	chunks                 []*chunk.Chunk
+	aggregators            []*Aggregator
+	dropFirstChunk         bool
+	ingestAfterNextChunkT0 uint32
```

Comment:
another benefit of my suggestion (see other comment) is if the user input is `ingestFrom`, then this var can be `ingestFromT0` and all the alignment here stays the same ;)

Reply:
> another benefit of my suggestion (see other comment) is if the user input is `ingestFrom`, then this var can be `ingestFromT0` and all the alignment here stays the same ;)

Implemented with 3bb1d5f